### PR TITLE
Feature stripe payment 70

### PR DESCRIPTION
**Pull request by garzeah**
### What this PR does (required):
- Added logic to process subscription payments with stripe
- Removed checkout page since stripe provides one

### Screenshots / Videos (required):
**Successful Transaction**
success-min

**Cancel Transaction**
cancel

### Any information needed to test this feature (required):
- Head to the upgrade page and try to upgrade

### Any issues with the current functionality (optional):
- Since we don't have any premium features, "buying" something doesn't really do anything.

### CODE REVIEW

File: server/app.js [hunk 1]
```diff
@@ -55,6 +56,8 @@ app.use("/auth", authRouter);
 app.use("/users", userRouter);
 app.use("/appointment", appointmentRouter);
 app.use("/meeting", meetingRouter);
+app.use("/availability", availabilityRouter);
```
Comment (ajaytomgeorge) [inline server/app.js]: May I know the purpose of having two different endpoints point to the same _availabilityRouter_. I can't see any other references to it

File: client/src/pages/Upgrade/Upgrade.tsx [hunk 2]
```diff
@@ -35,7 +35,7 @@ const Upgrade = (): JSX.Element => {
             headerSubtitle="$5/month"
             disableButton={false}
             buttonText="Upgrade"
-            redirectUrl="/checkout"
+            priceId="price_1JDNXxKWQLQuuRfnORSql3u5"
```
Comment (ajaytomgeorge) [inline client/src/pages/Upgrade/Upgrade.tsx]: Not a great fan of hardcoding ids in client side, If the PRs have a scope of using an API for priceId, you can use that

**Comment by ajaytomgeorge** [comment]
Great Work Overall. Just some minor comments.

File: server/app.js (hunk 1 above)
Comment (garzeah) [reply server/app.js]: Removed the duplicate, I forgot to remove this when resolving merge conflicts.

Comment (ajaytomgeorge) [reply server/app.js]: @bonnieli  Looks good from my side now

File: client/src/pages/Upgrade/Upgrade.tsx (hunk 2 above)
Comment (garzeah) [reply client/src/pages/Upgrade/Upgrade.tsx]: Updated! I just created a .ts file with list of subscriptions/priceIds so we can re-use it or add more payment plans in the future

**Review by moffatethan — APPROVED** [review]

**Pull request merged**